File: OBI - Aula/restaurante.c
#include <stdio.h>
#include<math.h>
#include <stdbool.h>

int main(void){
    float raio, largura, altura, diagonal;
    int testes=0;

    while(true){
        testes++;
        scanf("%f", &raio);
        if(raio==0) break;
        scanf("%f %f", &largura, &altura);

        diagonal = sqrt(pow((largura/2),2)+pow((altura/2),2));

        if(diagonal<=raio)  printf("Pizza %d fits on the table.\n", testes);
        else printf("Pizza %d does not fit on the table.\n", testes);
        
    }
}

File: OBI - Aula/restaurante.py
testes=0
while(True):
    testes+=1
    raio,largura,altura = [float(x) for x in input().split()]
    if raio==0:
        break
    
    diagonal = ((largura/2)**2+(altura/2)**2)**(1/2)

    if diagonal <=raio:
        print("Pizza %i fits on the table." %testes)
    else:
        print("Pizza %i does not fit on the table." %testes)